src/app/views/events-page/events-page.component.html

```diff
@@ -74,6 +94,39 @@
         </div>
     </div>
 
+    <div *ngIf="eventModal">
```

Comment:
I noticed that after I create an event, and it's successfully saved, if I click the "Schedule Event" button again, it retains all the input from the previous event I created.